### ISSUE TITLE
Adding warning on sharing with public 

### ISSUE THREAD

**Issue by simonfossom** [huridocs/uwazi#3573]
We need to add additional warning when sharing publicly.

![public](https://user-images.githubusercontent.com/8134850/113167145-ba3dab80-9243-11eb-9cd5-5e00af64d632.jpg)

([Link to design](https://www.figma.com/file/2AT2ilGWBTe0pWy4j66yxW/Uwazi-Access-Levels?node-id=835%3A939))

**Comment by fnocetti** [comment]
@simonfossom The alert is meant to be shown only when the user **adds** the permission right? It's not the case for when the permission was already there.

In both cases, this design doesn't take in count that there's a dropdown for the public permission two!

**Comment by simonfossom** [comment]
Yes, the warning is only when first added. Laster is like any other. 
About dropdown, there shouldn't be one. Public can only view entities.

**Comment by simonfossom** [comment]
Yes, you are right. I'm totally out of this one. Give a bit to think about it.

**Comment by fnocetti** [comment]
Finally, this is being addressed in #3570
